src/main/scala/com/databricks/spark/redshift/Parameters.scala

```diff
@@ -194,5 +195,12 @@ private [redshift] object Parameters extends Logging {
 
       s"aws_access_key_id=$accessKeyId;aws_secret_access_key=$secretAccessKey"
     }
+
+    /**
+     * When nonempty/non-null sets the compression codec to use for writing Avro data.
+     *
+     * Defaults to snappy.
```

Comment:
Default is actually disabled now, looks like this doc comment needs an update if we're keeping it disabled by default.